3p/ampcontext.js

```diff
@@ -41,30 +41,20 @@ export class AmpContext extends IframeMessagingClient {
   constructor(win) {
     super(win);
     this.setupMetadata_();
+    this.getHostWindow();
```

Comment:
Ah I wasn't thinking right, here. I need to make some changes to all the messaging out anyways, and I will fix as per your comment as part of that